### DISCUSSION TITLE
How to add MenuBarExtraAccess to a multiplatform App?

### DISCUSSION THREAD

**Discussion by meixuesong** [orchetect/MenuBarExtraAccess#8] [Q&A] [answered]
Hi, 
I'm developing a multiplatform app (iOS & macOS), I need MenuBarExtraAccess for the macOS app, but if I add the dependency through Swift Package Manager, it will fail when build the iOS app. 

The error message is: `No such module 'AppKit'` for line 7 of MenuBarExtraUtils.swift.

So how can I add MenuBarExtraAccess to a multiplatform app? 

Thanks.

**Comment by orchetect** [comment]
I can update the package so it builds on all platforms. I might be able to get that done today - I'll let you know when there's a release.

**Reply by orchetect** [reply, marked as answer]
@meixuesong Release 1.0.5 is up that should build for all platforms.

**Reply by meixuesong** [reply]
great! Thank you @orchetect
